infrastructure/kube/keep-prd/bitcoin/bitcoind/bitcoind-volumesnapshot.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+  name: bitcoind-snapshot
+spec:
+  volumeSnapshotClassName: bitcoind
+  source:
+    persistentVolumeClaimName: bitcoind-data-bitcoind-0
```

Comment:
Why is it just for `-0`? From the PR, I understand that we first synced one instance, then used the snapshots to sync the second instance faster. Long-term, don't we want to have snapshots for both PVCs? If `-0` gets corrupted for whatever reason, we could restore it from `-1`.

Reply:
We needed snapshot from `bitcoind-data-bitcoind-0` to spin up the second replica. There was no point of creating a snapshot of `bitcoind-data-bitcoind-1`, as they would be pretty close. After some time I plan to take a new snapshot and repeat it periodically.